todo/src/actions/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+let newToDoId = 0;
+
+const addToDo = todo => ({
+  type: "ADD_TODO",
+  value: todo,
+  id: newToDoId++
```

Comment:
```
id: newToDoId++
```
could be replaced by using something like
```
key={todo.id}
```
inside the parent element rendering the todos 👍 
Yours is creative and works though. 👌